dwave/system/testing.py

```diff
@@ -65,21 +62,28 @@ class MockDWaveSampler(dimod.Sampler, dimod.Structured):
             Supported options are 'chimera', 'pegasus' or 'zephyr'.
             The default is 'chimera' when the value is not specified as part 
             of the from the ``properties`` argument.
-            
+
         topology_shape (string, optional):
             A list of three numbers [m,n,t] for Chimera, defaulted as [4,4,4]. 
             A list of one number [m] for Pegasus, defaulted as [3].
             A list of two numbers [m,t] for Zephyr, defaulted as [2,4].
             The default above apply only when the value is not 
             specified as part of the from the ``properties`` argument.
-            
+
         parameter_warnings (bool, optional, default=True):
             The MockSampler is adaptive with respect to ``num_reads``,
             ``answer_mode`` and ``max_answers`` and ``label`` 
             parameters. By default ``initial_state`` can also be mocked, if
             dwave-greedy is installed. All other parameters are ignored and a 
             warning will be raised by default.
 
+        exact_solver_cutoff (int, optional, default=:attr:`EXACT_SOLVER_CUTOFF_DEFAULT`):
+            Maximum problem size for which the :class:`~dimod.ExactSolver` is used in
```

Comment:
It's not clear in the current docstring how calculation of ground state energy impacts the samplesets returned by the solver. It might be worth explicitely implementing an upper bound so that users don't get strange errors from ExactSolver, or large unexpected delays. Something like:

For problems smaller than ``exact_solver_cutoff`` the first sample in any sampleset returned by the class sampling routines is replaced by a reproducible ground state (determined exactly). To disable this feature set ``exact_solver_cutoff=0``. Only small cut offs are supported since time resources scale exponentially in system size.

It's not necessary to get into specific details about how that is done (such as use of ExactSolver) IMO.